### PR TITLE
Add section about adyen notification system

### PR DESCRIPTION
**Pull request by korycins**

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1249 Add paragraph about enabling the Adyen's notifications](https://app.clickup.com/t/2549495/SALEOR-1249)

File: docs/dashboard/configuration/plugins.md [hunk 1]
```diff
@@ -66,8 +66,17 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
+- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
 
 Below you can find an example configuration of the Adyen plugin:
 
 ![Adyen](../screenshots/config-plugins-adyen.png)
+
+### Activating the Adyen notification
+
+Adyen uses a notification system that can be used to update the payment information on the Saleor side.  Enabling the notification requires some additional steps on the Adyen's customer area panel side. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area) for more details.
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:77]: Can't tell if this is required for Saleor to work or optional. Can we provide step-by-step instructions? Even if it's as simple as "step 1: follow this Adyen doc, step 2: use this value as the callback URL, step 3: here's how to verify that it works"?

File: docs/dashboard/configuration/plugins.md [hunk 2]
```diff
@@ -66,8 +66,17 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
+
+The URL that you need provide on Adyen side is `https://<your-backend-url>/plugins/mirumee.payments.adyen/webhooks/`.
+
+
+To protect your notification endpoints make sure that you enabled the HMAC signature and basic authentication [Adyen docs about security]([https://docs.adyen.com/development-resources/webhooks/best-practices#security]). [See Saleor configuration fields](#enabling-the-adyen-integration)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:82]: I think this should be part of the step-by-step instructions above: once you verify that Adyen works, enable payload signing and check again.

**Review by maarcingebala — APPROVED** [review]

File: docs/dashboard/configuration/plugins.md [hunk 3]
```diff
@@ -66,8 +66,22 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
+1. Follow [the Adyen docs about notifications.](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)
+    * for `URL` use `https://<your-backend-url>/plugins/mirumee.payments.adyen/webhooks/`.
+    * for `SSL Version` use the newest one.
+    * for `Method` use JSON.
+2. To protect your notification follow [the Adyen docs about security.](https://docs.adyen.com/development-resources/webhooks/best-practices#security)
+    * copy HMAC key and paste it to Saleor field **HMAC secret key**. [(See Saleor configuration fields.)](#enabling-the-adyen-integration)
+    * copy the username from _Authentication_ section and paste it to Saleor field **Notification user**. [(See Saleor configuration fields.)](#enabling-the-adyen-integration)
+    * copy the password from _Authentication_ section and paste it to Saleor field **Notification password**. [(See Saleor configuration fields.)](#enabling-the-adyen-integration)
+3. To test your configuration check the [the Adyen docs about testing the configuration.](https://docs.adyen.com/development-resources/webhooks#test-your-notifications-server)
```
Comment (karolkielecki) [inline docs/dashboard/configuration/plugins.md:87]:
```suggestion
3. To test your configuration check [the Adyen docs about testing the configuration.](https://docs.adyen.com/development-resources/webhooks#test-your-notifications-server)
```

**Review by karolkielecki — CHANGES_REQUESTED** [review]
Line 87 should be updated

**Review by karolkielecki — APPROVED** [review]

**Comment by korycins** [comment]
I've updated the docs and screenshot after we added `Enable native 3d secure` and dropped the origin field.

File: docs/dashboard/configuration/plugins.md [hunk 4]
```diff
@@ -66,8 +64,24 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
 
-- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [`Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
+- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
+
+- **Enable native 3d secure:** Saleor uses 3D Secure redirect authentication by default. If you want to use native 3D Secure authentication, enable this option. For more details see Adyen documentation: [native-3ds2](https://docs.adyen.com/checkout/3d-secure/native-3ds2), [redirect-3ds2-3ds1](https://docs.adyen.com/checkout/3d-secure/redirect-3ds2-3ds1)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:69]: It's either "3-D Secure" (the official name) or "3D Secure". Please use the proper name here and in the actual UI.

File: docs/dashboard/configuration/plugins.md [hunk 5]
```diff
@@ -66,8 +64,24 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
+### Activating the Adyen notification
+
+
+Adyen uses a notification system. Enabling it is required for proper work of payment gateway.
+1. Follow [the Adyen docs about notifications.](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)
+    * for `URL` use `https://<your-backend-url>/plugins/mirumee.payments.adyen/webhooks/`.
+    * for `SSL Version` use the newest one.
+    * for `Method` use JSON.
+2. To protect your notification follow [the Adyen docs about security.](https://docs.adyen.com/development-resources/webhooks/best-practices#security)
+    * copy HMAC key and paste it to Saleor field **HMAC secret key**. [(See Saleor configuration fields.)](#enabling-the-adyen-integration)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:84]:
```suggestion
    * Copy the HMAC key into the Saleor's **HMAC secret key** field. [(See Saleor configuration fields.)](#enabling-the-adyen-integration)
```

File: docs/dashboard/configuration/plugins.md [hunk 6]
```diff
@@ -66,8 +64,24 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
 Below you can find an example configuration of the Adyen plugin:
 
 ![Adyen](../screenshots/config-plugins-adyen.png)
+
+### Activating the Adyen notification
+
+
+Adyen uses a notification system. Enabling it is required for proper work of payment gateway.
+1. Follow [the Adyen docs about notifications.](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)
+    * for `URL` use `https://<your-backend-url>/plugins/mirumee.payments.adyen/webhooks/`.
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:80]: Start with a capital letter when ending the sentence with a dot.

File: docs/dashboard/configuration/plugins.md [hunk 7]
```diff
@@ -66,8 +64,24 @@ Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
...
 
-- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [`Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
+- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
+
+- **Enable native 3D secure:** Saleor uses 3D Secure redirect authentication by default. If you want to use native 3D Secure authentication, enable this option. For more details see Adyen documentation: [native 3D Secure 2](https://docs.adyen.com/checkout/3d-secure/native-3ds2), [redirect 3D Secure 1 and 2](https://docs.adyen.com/checkout/3d-secure/redirect-3ds2-3ds1)
```
Comment (patrys) [inline docs/dashboard/configuration/plugins.md:69]:
```suggestion
- **Enable native 3D Secure:** Saleor uses 3D Secure redirect authentication by default. If you want to use native 3D Secure authentication, enable this option. For more details see Adyen documentation: [native 3D Secure 2](https://docs.adyen.com/checkout/3d-secure/native-3ds2), [redirect 3D Secure 1 and 2](https://docs.adyen.com/checkout/3d-secure/redirect-3ds2-3ds1)
```

**Review by maarcingebala — APPROVED** [review]

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**